src/map/lua/lua_baseentity.cpp

```diff
@@ -2668,6 +2668,11 @@ void CLuaBaseEntity::setPos(sol::variadic_args va)
                 return;
             }
 
+            if (((CCharEntity*)m_PBaseEntity)->PPet != nullptr)
```

Comment:
Just for consistency, can the C-style casts please be converted to static_cast?

Reply:
Done. Touched up the code in the pet respawning section while at it, just so the block all matches.